src/Theme.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+// "Theme" is a configurable set of CET ImGui styling parameters
+//
+// NB: this is not a serialization of ImGuiStyle, it's a
+//     serializable representation of styling parameters.
+
+// Load ImGui style directly from JSON. Parses the format used by DumpStyleToThemeJson.
+bool LoadStyleFromThemeJson(const std::filesystem::path& aThemePath, ImGuiStyle& aOutStyle);
+
+// Dump ImGui style into a JSON string with some extra metadata. Can be saved.
+const std::string DumpStyleToThemeJson(const ImGuiStyle& aStyle);
```

Comment:
i would just return std::string, returning const std::string would just lead to unnecessary copies (and would make more sense if you returned reference here but you do not, so std::string is better return value in this case. allows for copy ellision)